text/src/typography.rs

```diff
@@ -359,10 +344,15 @@ where
   }
 
   fn is_over_line_bound(&self, position: Em) -> bool {
+    if self.cfg.text_align == TextAlign::Center {
+      return false;
```

Comment:
Why is it possible to just return false in this case?

Reply:
When TextAlign is Center, the entire line needs to be laid out to determine the position, so the layout boundary is ignored